MetaMask/metamask-extension
# feat: Add header info icon

## **Description**
As part of the redesign task, we need to add an info icon to the header that displays the sender (from) address and balance in a modal when clicked. 

[See design](https://www.figma.com/file/uACzbBjwU2tV30UuewYiTx/Confirmation-redesign-V2?type=design&node-id=982-34677&mode=design&t=IanteD1CJ5MeHToB-0)

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23366?quickstart=1)

## **Related issues**

Fixes: #21280 

## **Manual testing steps**

1. Add `- conf-redesign` to build types in builds.yml
2. Build MetaMask
3. Open Testdapp and click Sign in Personal Sign Section
4. Screenshots as shown in the design should be shown

## **Screenshots/Recordings**

<img width="362" alt="Screenshot 2024-03-07 at 14 30 38" src="https://github.com/MetaMask/metamask-extension/assets/44811/396e4679-5dd5-45c8-8c2f-b720db84e386">

<img width="375" alt="Screenshot 2024-03-07 at 14 30 55" src="https://github.com/MetaMask/metamask-extension/assets/44811/4d8e8734-3a9c-4c53-9b3c-8a3f51144411">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
